debezium-server/debezium-server-redis/pom.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.debezium</groupId>
+        <artifactId>debezium-server</artifactId>
+        <version>1.5.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>debezium-server-redis</artifactId>
+    <name>Debezium Server Redis Sink Adapter</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <skipITs>true</skipITs>
```

Comment:
This should should be removed as the test can run as part of CI/regular build as the test is self-contiane din the same way as PulsarIT.

Reply:
@jpechane Could you please help me to resolve following error?

> Caused by: io.debezium.DebeziumException: No Debezium consumer named 'test' is available

Reply:
You need to provide file like https://github.com/debezium/debezium/blob/master/debezium-server/debezium-server-pulsar/src/test/resources/META-INF/services/org.eclipse.microprofile.config.spi.ConfigSource

Please check Pulsar module for project structure. In case the image will need to be modified then you can look there as well.